Resources/doc/reference/action_export.rst

```diff
@@ -12,6 +12,12 @@ This document will cover the Export action and related configuration options.
 Basic configuration
 -------------------
 
+Translation
+~~~~~~~~~~~
+
+All field names are translated by default.
+An internal mechanism checks if a field matching the translator strategy label exists in the current translation file and will use the field name as a fallback.
```

Comment:
Line 19 is longer than 120 characters, that's the reason for linebreaking. We do it for code because it's more readable, but text is also meant to be read.